Update static-outbound-ip-addresses.md (#7960)
umbraco/UmbracoDocs: umbraco-cloud/expand-your-projects-capabilities/external-services/static-outbound-ip-addresses.md

Umbraco Cloud services access external applications using static outbound IP addresses. This enables you to allowlist Cloud services in IP-based firewalls. This is particularly useful if you wish to control access to your website based on IP addresses.

{% hint style="warning" %}

Changing plans (topology changes)

> When you change your Umbraco Cloud project plan, such as moving from Shared to Dedicated or between Dedicated tiers, the static outbound IP addresses may not be applied for a short time. During this period, outbound traffic from your project may come from an IP address outside your static outbound IP range.

Reviewer: Still a bit too long this one.
Here's a suggestion:

```suggestion
Moving between Umbraco Cloud plans, like switching from Shared to Dedicated tiers, may briefly disrupt static outbound IP addresses. They might not apply for a short time. During this period, outbound traffic from your project may come from an IP address outside your static outbound IP range.
```

> If your solution connects to external services that allow traffic based on IP address, such as external SQL databases or third party APIs, this can cause connection failures during and shortly after the plan change.

Reviewer: Same here: 

```suggestion
Connections to IP-restricted external services, like SQL databases or APIs, may fail during and shortly after a plan change.
```
